recipes/edgepro/meta.yaml

```diff
@@ -0,0 +1,27 @@
+build:
+  number: 0
+
+about:
+    home: http://ccb.jhu.edu/software/EDGE-pro/index.shtml
+    license: compliant with https://opensource.org/docs/definition.php
+    summary: EDGE-pro (Estimated Degree of Gene Expression in PROkaryotes) is an efficient software system to estimate gene expression levels in prokaryotic genomes from RNA-seq data. EDGE-pro uses Bowtie2 for alignment and then estimates expression directly from the alignment results. EDGE-pro includes routines to assign reads aligning to overlapping gene regions accurately. 15% or more of bacterial genes overlap other genes, making this a significant problem for bacterial RNA-seq, one that is generally ignored by programs designed for eukaryotic RNA-seq experiments. 
+
+package:
+    name: edge-pro
+    version: 1.3.1
+source:
+    url: http://ccb.jhu.edu/software/EDGE-pro/EDGE_pro_v1.3.1.tar.gz
+    fn: EDGE_pro_v1.3.1.tar.gz
+    sha256: 70fe107459703ba407d3555acfa9a64a8a0334a7
```

Comment:
sha256 should be fe6e6bbd8a74294de25d6df1add76e2aa6df0d8df6e450b5905fa9bdbcae9e4e

Reply:
oh yes, I used wrong option. Thanks for spotting